responses.py

```diff
@@ -692,6 +692,23 @@ def stop(self, allow_assert=True):
                 )
             )
 
+    def assert_called_once(self, url):
```

Comment:
Would getting the call count for a URL give a more general purpose API? For example you'd be able to assert 0, or N calls with the same amount of code as 1.

Reply:
@markstory, Thanks for your suggestion.
I've changed it to accept `call_count`.
Please let me know if the function name or documentation needs changes.